### DISCUSSION TITLE
Super Quickhop

### DISCUSSION THREAD

**Discussion by ForTimeBeing** [runelite/runelite#16661] [Ideas]
OSRS is a game set in an open world, where the availability of resources is limited and demands the establishment of a personal space to efficiently engage in specific activities. However, players often face the repetitive task of hopping between worlds to find an open area.

Here's an exciting proposition: envision a feature within Runelite that seamlessly examines fellow players in your vicinity within worlds. When you initiate a quickhop, this functionality scans for other Runelite users within that chunk in the subsequent world. Should it locate such players, the quickhop conveniently bypasses that particular world and tries the next one.

Implementing this feature would greatly enhance the ease and alleviate the frustration associated with quick hopping.

**Comment by LlemonDuck** [comment]
This would be extremely resource-consuming on the server side, and would also be easily spoofable by bad actors to guarantee their world isn't taken while they reset / are preparing to go.

**Reply by ForTimeBeing** [reply]
Yeah this was my worry. I was hoping maybe somone had a better idea to optimize it in some way. It would be a super useful feature all players would use
